examples/stocks/lib/stock_settings.dart

```diff
@@ -225,7 +225,7 @@ class StockSettingsState extends State<StockSettings> {
           )
         ),
         new DrawerItem(
-          icon: Icons.gradient,
+          icon: new Icon(Icons.gradient),
```

Comment:
I thought of doing that but what color or size should it give? If we give nothing, then how do you configure an icon? Do you always give an IconTheme and rely on inheritance?

In the end I went with what you see here. (I also considered `DrawerItem.icon` as a constructor, but it looked really ugly when I tried using it.)